kong/1.9/README.md

```diff
@@ -1,112 +1,203 @@
+    - `contact-points`: Comma separated list of Cassandra contact points.
+    - `port`: Port on which Cassandra listening for query.
+    - `keyspace`: Keyspace to use in Cassandra. Will be created if doesn't exist.
 
-- `username`: This parameter configures the username for the kong database.
-- `password`: This parameter configures the password for the kong database.
-- `dbname`: This parameter configures the name of the kong database.
-- `persistence`: This parameter enables persistent volumes for postgresql.
+    Install Kong using `kong-cassandra.json` file:
 
-Install postgresql using this `options.json` file:
+    ```bash
+    $ dcos package install kong --options=kong-cassandra.json
+    ```
 
-```bash
-$ dcos package install postgresql --options=options.json
-```
+5. **Verify your deployments**
 
-To verify that our postgresql instance is up and running, we can use `dcos task` command:
+    To verify that our kong instance is up and running, we can use `dcos task`
+    command:
 
-```bash
-$ dcos task
-NAME         HOST        USER  STATE  ID
-marathon-lb  10.0.6.130  root    R    marathon-lb.104e0ebe-03be-11e7-b790-627a33830456
-postgresql   10.0.0.71   root    R    postgresql.0f2867a0-03cb-11e7-b790-627a33830456
-```
-## Install Kong
-Next, install Kong. If you have installed postgresql with a different username, password, dbname, or id, customize the kong installation accordingly.  
-```bash
-$ dcos package install kong
-This DC/OS Service is currently EXPERIMENTAL. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Prerequisites for this service include postgres and marathon-lb. See https://github.com/dcos/examples/tree/master/1.8/kong for more details.
-Continue installing? [yes/no] yes
-Installing Marathon app for package [kong] version [0.9.9]
-Kong has been installed.
-```
-To verify that our kong instance is up and running, we can use `dcos task` command:
+    ```bash
+    $ dcos task
+    NAME         HOST        USER  STATE  ID
+    kong         10.0.1.8   root    R    kong.af46c916-3b55-11e7-844e-52921ef4378d         
+    marathon-lb  10.0.4.42  root    R    marathon-lb.d65c3cc3-3b54-11e7-844e-52921ef4378d  
+    postgres     10.0.1.8   root    R    postgres.5b0a2635-3b55-11e7-844e-52921ef4378d   
+    ```
 
-```bash
-$ dcos task
-NAME         HOST        USER  STATE  ID
-kong         10.0.0.71   root    R    kong.c2cd2cc9-03cb-11e7-b790-627a33830456
-marathon-lb  10.0.6.130  root    R    marathon-lb.104e0ebe-03be-11e7-b790-627a33830456
-postgresql   10.0.0.71   root    R    postgresql.0f2867a0-03cb-11e7-b790-627a33830456
-```
+    Kong in the DC/OS UI
 
-Kong in the DC/OS UI
+    ![Kong on DC/OS](img/kong-dcos.png)
+  
+6. **Using Kong**
+    
+    Now that Kong is installed, curl the marathon-lb endpoint to validate that
+    it is up and running:
 
-![Kong on DC/OS](img/kong-dcos.png)
+    ```bash
+    $ curl <public-agent-ip>:10201
```

Comment:
update the example to say that this is a valid response.
And where does the user run Kong command? Master, node where its installed?